### ISSUE TITLE
Mozilla Persona is closed, easy login strategy for testing?

### ISSUE THREAD

**Issue by Frintrop** [NREL/api-umbrella#323]
Mozilla Persona is closed. Is there any easy login strategy for testing? Tried google, but getting invalid credentials connecting to 127.0.0.1/admin ......

**Comment by Frintrop** [comment]
see #314

**Comment by GUI** [comment]
Yeah, we are working on a local authentication replacement that will be part of the next release (current work is in the [local-admin-accounts](https://github.com/NREL/api-umbrella/tree/local-admin-accounts) branch).

In the meantime, yeah, unfortunately the other login options require a bit more config. If you're just interested in testing purposes, then one workaround would be to set your `/etc/api-umbrella/api-umbrella.yml` config file to contain:

```yml
app_env: development
```

This will run things in development mode, which will be a bit slower, but enables a "dummy" login option for the admin where you can just enter any e-mail address you want and you'll be logged in. So obviously this isn't intended for real use, but that might be the quickest current way to play around locally.

**Comment by GUI** [comment]
FYI: v0.14.0 is released which has local admin accounts, which should be easier to setup.
